locales/hu/messages.json

```diff
@@ -0,0 +1,128 @@
+    "message": "A weboldalak szövegtérköz-tulajdonságainak szerkesztésére szolgáló eszköz, ami WCAG megfelelőséggel kapcsolatos információkat is tartalmaz."
+  },
+  "toggleCommand": {
+    "message": "Az egyéni szövegtérköz-tulajdonságok engedélyezése vagy letiltása.",
+    "description": "A billentyűkombináció leírása a böngészőben."
+  },
+  "buttonSetWCAGValues": {
+    "message": "WCAG értékek beállítása"
+  },
+  "buttonResetAllValues": {
+    "message": "Összes érték visszaállítása"
+  },
+  "valueDefault": {
+    "message": "alapértelmezett",
+    "description": "A szövegtérköz-tulajdonság alapértelmezett értéke."
+  },
+  "lineHeight": {
+    "message": "Sorköz"
+  },
+  "letterSpacing": {
+    "message": "Betűköz"
+  },
+  "wordSpacing": {
+    "message": "Szavak közti távolság"
+  },
+  "paragraphSpacing": {
+    "message": "Bekezdések közti távolság"
+  },
+  "active": {
+    "message": "Aktív"
+  },
+  "decreasePropertyValue": {
+    "message": "$PROPERTY$ értékének csökkentése",
+    "placeholders": {
+      "property": {
+        "content": "$1"
+      }
+    }
+  },
+  "increasePropertyValue": {
+    "message": "$PROPERTY$ értékének növelése",
+    "placeholders": {
+      "property": {
+        "content": "$1"
+      }
+    }
+  },
+  "resetPropertyValue": {
+    "message": "$PROPERTY$ értékének visszaállítása",
+    "placeholders": {
+      "property": {
+        "content": "$1"
+      }
+    }
+  },
+  "propertiesWereEnabledDisabled": {
+    "message": "Az egyedi szövegtérköz-tulajdonságok $STATUS$.",
+    "placeholders": {
+      "status": {
+        "content": "$1"
+      }
+    }
+  },
+  "enabled": {
+    "message": "engedélyezve lettek"
+  },
+  "disabled": {
+    "message": "le lettek tiltva"
+  },
+  "propertiesWereSetToWCAG": {
+    "message": "Az egyedi szövegtérköz-tulajdonságok a WCAG értékekre lettek beállítva."
+  },
+  "propertiesWereResetToDefaults": {
+    "message": "Az egyedi szövegtérköz-tulajdonságok az alapértelmezett értékeikre lettek visszaállítva."
+  },
+  "propertyWasResetToDefault": {
+    "message": "Az egyedi $PROPERTY$ tulajdonság az alapértelmezett értékére lett visszaállítva.",
+    "placeholders": {
+      "property": {
+        "content": "$1"
+      }
+    }
+  },
+  "instructionsForUsage": {
+    "message": "Hogyan használja a Szövegtérköz-szerkesztőt?"
+  },
+  "instructionsForActive": {
+    "message": "A felül lévő <strong class='hint'>Aktív</strong> kapcsoló segítségével aktiválhatja, illetve kikapcsolhatja az Ön által beállított egyedi szövegtérköz-tulajdonságokat, valamint a bővítmény interaktív vezérlőit. Ugyanehhez a funkcióhoz a <code class='hint'>Ctrl+Shift+F</code> (Mac-en a <code class='hint'>Command+Shift+F</code>) billentyűkombinációt is használhatja. (Igény szerint egyéni billentyűkombináció is megadható a böngésző bővítmény beállításaiban.) Az egyéni tulajdonságok átkapcsolásának lehetősége praktikus az egyéni beállítások teszteléséhez és tárolásához."
```

Comment:
In here I would not mix the "activate - deactivate" and "turning off - turning on" pairs. If there is "activate" in the first part of the sentence I would use "deactivate" after the comma as well. So in my opinion it should be either:

"A felül lévő <strong class='hint'>Aktív</strong> kapcsoló segítségével aktiválhatja, illetve deaktiválhatja az Ön által beállított egyedi szövegtérköz-tulajdonságokat..."

OR

"A felül lévő <strong class='hint'>Aktív</strong> kapcsoló segítségével bekapcsolhatja, illetve kikapcsolhatja az Ön által beállított egyedi szövegtérköz-tulajdonságokat..."

Reply:
Fixed